services/users-api/lambdas/fetchReferenceCode.js

```diff
@@ -0,0 +1,82 @@
+import to from 'await-to-js';
```

Comment:
I was wondering if we should make a completely new api service which is only used for admins using the office add-in?

Reply:
I agree, right now this lambda feels a bit out of place